frontend/public/components/events.jsx

```diff
@@ -182,6 +184,11 @@ class EventsStreamPage_ extends React.Component {
 
 export const EventStreamPage = connectToFlags(FLAGS.OPENSHIFT, FLAGS.PROJECTS_AVAILABLE)(EventsStreamPage_);
 
+const measurementCache = new CellMeasurerCache({
+  fixedWidth: true,
+  minHeight: 109, /* height of event with a one-line event message on desktop */
```

Comment:
Not sure if we should not use **em** units since they are scalable in nature and we are hardcoding the value here.

Reply:
I don't see a way to use different units with react-virtualized.

Reply:
Makes sense :+1: